webpack/assets/javascripts/react_app/components/Editor/components/__tests__/EditorSettings.test.js

```diff
@@ -27,5 +34,21 @@ const fixtures = {
   },
 };
 
-describe('EditorSettings', () =>
-  testComponentSnapshotsWithFixtures(EditorSettings, fixtures));
+describe('EditorSettings', () => {
```

Comment:
This test no longer renders a snapshot of all the elements in editor settings, please use react-testing-library to ensure all the elements are shown correctly. rendering using `mount` instead of `shallow` creates a 1.5k lines of snapshots so we cant just change it and leave the test as it is.